src/main/java/com/pinterest/secor/uploader/Uploader.java

```diff
@@ -47,16 +47,17 @@
     private ZookeeperConnector mZookeeperConnector;
     private UploadManager mUploadManager;
 
-    public Uploader(SecorConfig config, OffsetTracker offsetTracker, FileRegistry fileRegistry,
-                    UploadManager uploadManager) {
-        this(config, offsetTracker, fileRegistry, uploadManager,
-             new ZookeeperConnector(config));
+
```

Comment:
Will do, even though there was no JavaDoc on the previous constructor ... but I can deduct what is what from the code :-)